packages/flask/src/UICountryPicker/CountryPicker.tsx

```diff
@@ -0,0 +1,238 @@
+        }),
+        [insets?.bottom],
+    );
+
+    const onSelectCountry = React.useCallback(
+        (item: Country) => {
+            onSelect && onSelect(item.code);
+        },
+        [onSelect],
+    );
+
+    const checkIncludes = React.useCallback((code: string) => {
+        let isPermitted = true;
+        let isBanned = false;
+        if (permitted.length) {
+            isPermitted = permitted.includes(code);
+        }
+        if (banned.length) {
+            isBanned = banned.includes(code);
+        }
+        return isPermitted && !isBanned;
+    }, []);
+
+    const filterCountries = React.useCallback(
+        list => {
+            const permittedCountries = list.filter((country: any) => checkIncludes(country.code));
+            setFilteredList(permittedCountries);
+        },
+        [banned, permitted],
+    );
+
+    React.useEffect(() => {
+        setFilteredList(countriesList);
+    }, [countriesList]);
+
+    React.useEffect(() => {
+        const result = search ? fuse.search(search) : countriesList;
+        setFilteredList(result as CountriesArray);
+    }, [search]);
+
+    React.useEffect(() => {
+        fetchJSON()
+            .then((r: CountriesArray) => {
+                if (permitted.length || banned.length) {
+                    filterCountries(r);
+                } else {
+                    setCountriesList(r);
+                }
+            })
+            .catch((e: Error) => {
+                console.error(e);
+            })
+            .finally(() => {
+                setLoading(false);
+            });
+    }, []);
+
+    const renderSearchHeader = () => {
+        return (
+            <View style={styles.headerContainer}>
+                <View style={styles.headerTitleContainer}>
+                    <View style={styles.sideHeaderView}>
+                        <UILinkButton onPress={onClose} title={uiLocalized.Cancel} />
+                    </View>
+                    <UILabel role={TypographyVariants.HeadlineHead} style={styles.headerTitle}>
+                        {uiLocalized.CountryPicker.Title}
+                    </UILabel>
+                    <View style={styles.sideHeaderView} />
+                </View>
+                <UISearchBar returnKeyType="done" value={search} onChangeText={setSearch} />
+            </View>
+        );
+    };
+
+    const renderCountryRow = ({ item }: { item: Country }) => {
+        const onPress = () => onSelectCountry(item);
+        return (
+            <TouchableOpacity onPress={onPress} style={styles.rowContainer}>
+                <View style={styles.rowContainerInner}>
+                    <UILabel>{item.name}</UILabel>
+                    <UILabel style={styles.emojiContainer}>{item.emoji}</UILabel>
+                </View>
+            </TouchableOpacity>
+        );
+    };
+
+    const renderLoading = () => {
+        return <ActivityIndicator />;
```

Comment:
Can you please use [UIIndicator](https://github.com/tonlabs/UIKit/blob/development/packages/hydrogen/src/UIIndicator.tsx) instead